src/textarea/textarea.test.tsx

```diff
@@ -96,4 +96,47 @@ describe('Textarea', () => {
 
         expect(textarea.disabled).toBeTruthy();
     });
+
+    it('check for character limit circle', () => {
+        const id = 'my-txtarea';
+        const value = 'Textarea content';
+        const onChange = jest.fn();
+
+        const { getByDataQa } = render(
+            <Textarea id={id} enableCharacterLimit={true} value={value} onChange={onChange} />
+        );
+        const characterLimitArea = getByDataQa(`character-limit-circle`) as HTMLTextAreaElement;
```

Comment:
No impact but I think it's not of type `HTMLTextAreaElement`

Reply:
fixed